### ISSUE TITLE
multiple predicates in query results in query compilation error

### ISSUE THREAD

**Issue by cdmistman** [smacker/go-tree-sitter#110]
using the following query:
```lisp
(import_statement
	source: (string) @import)
(
	(call_expression
		function: (identifier) @function
		arguments: (arguments ((_) @other-imports)? ((_) @import) .))
	(#eq? @other-imports "")
	(#eq? @function "require"))
```

results in the following error:
```
wrong number of arguments to `#eq?` predicate. Expected 2, got 6 
```

i've tested this using the tree-sitter cli with the JS language and it works. note that the issue appears only when both `eq?` predicates are present in the 2nd pattern.

**Comment by joshlarsen** [comment]
Looks like this would be fixed in #109

**Comment by smacker** [comment]
The PR has been merged. Please try again and feel free to re-open the issue if the problem was not fixed.
